lint.sh

```diff
@@ -3,25 +3,33 @@
 set -e
 
 # Global options
-declare -a OPTIONS=( -x ANSIBLE0011 --exclude=roles/coreos-bootstrap )
+declare -a OPTIONS=( --exclude=roles/coreos-bootstrap )
+declare -a IGNORES=( ANSIBLE0011 )
 
-# Per-file options
-declare -A F_OPTIONS=(
-    [shell.yml]="-x ANSIBLE0006"
-    [ldap_ban.yml]="-x ANSIBLE0012"
+# Per-file lint ignores
+declare -A F_IGNORES=(
+    [shell.yml]="ANSIBLE0006"
+    [ldap_ban.yml]="ANSIBLE0012"
 )
 
-exec() {
+function exec() {
```

Comment:
This one still has parenthesis where the `function join` below does not &lt;/nit&gt;

Reply:
Indeed!
Send a PR?  :D